.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+.idea
```

Comment:
If you'd like to ignore things that are very specific to your developer environment you should look into creating a .global_gitignore file in your home directory, otherwise we'll end up with things specific to lots of development environments here.